build/deps/mongo.Dockerfile

```
@@ -0,0 +1 @@
+FROM ghcr.io/ferretdb/mongo:6.0.3
```

```suggestion
FROM ghcr.io/ferretdb/legacy-mongo-shell:6.0.3-2
```

See https://github.com/FerretDB/deps/pkgs/container/legacy-mongo-shell